src/client/components/HealthCareApp.jsx

```diff
@@ -94,11 +107,16 @@ class HealthCareApp extends React.Component {
     const path = this.props.location.pathname;
     const formData = this.props.data;
     const sectionFields = this.props.uiState.sections[path].fields;
+    const noScrollPages = ['/household-information/financial-disclosure', '/household-information/annual-income'];
```

Comment:
Since the required questions are at the very bottom of these pages, it prevents the screen from scrolling to the top and obscuring them.